assets/vue/convos-dialog-settings.vue

```diff
@@ -25,6 +25,14 @@
           </a>
         </div>
       </div>
+      <div class="row" v-if="settings.sharelog_enabled">
+        <div class="col s12 li-link">
+          <a href="#" target="_blank">
+            <i class="material-icons">share</i>
+            Share conversation
+          </a>
+        </div>
+      </div>
```

Comment:
I think I can write a AssetPack pipe, which manipulates the vue assets. The only downside with that is that you *might* need optional modules for a plugin to work.